ops/model.py

```diff
@@ -2662,9 +2660,9 @@ class RelationDataTypeError(RelationDataError):
 
 
 class RelationDataAccessError(RelationDataError):
-    """Raised by ``Relation.data[entity][key] = value`` if you don't have access.
+    """Raised by ``Relation.data[entity][key] = value`` if sufficient access is not available.
```

Comment:
Perhaps simplify/directify:

```suggestion
    """Raised by ``Relation.data[entity][key] = value`` if unable to access.
```